components/region_cache_memory_engine/src/background.rs

```diff
@@ -446,6 +447,23 @@ impl BackgroundRunnerCore {
             .write()
             .mut_range_manager()
             .on_delete_ranges(ranges);
+        #[cfg(test)]
+        flush_epoch();
+    }
+}
+
+// flush epoch and pin enough times to make the delayed operations be executed
+#[cfg(test)]
+pub(crate) fn flush_epoch() {
+    // Epoch is global unique, so sleep a while to mitigate the influence of unpined
+    // guard in other concurrent test cases.
+    std::thread::sleep(Duration::from_secs(2));
+    {
+        let guard = &epoch::pin();
+        guard.flush();
+    }
+    for _ in 0..128 {
+        let _ = &epoch::pin();
```

Comment:
What's the point to have a tight loop for epoch::pin()? Can you add comment for this?

Reply:
Done

Reply:
Is the passive call with 128 pins necessary for ut? Maybe redundant ?

The previous `guard.flush()` should flush all older epoch.

Reply:
only pin 128 times will local epoch try to advance the global epoch and after that, the flushed operation will be executed.

Reply:
only pin 128 times will local epoch try to advance the global epoch and after that, the flushed operation will be executed.